addon/styles/_frost-text.scss

```diff
@@ -15,7 +15,6 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
-  width: 175px;
```

Comment:
I guess it doesn't prevent consuming context from overriding, but the same can be said for the `max-width` that was already removed, right? If you don't want to make people override that, why do you want to make people override this? Or is the argument that fewer people would want to override `width` than would want to override `max-width` or `min-width`?